src/transpose/testing.js

```diff
@@ -0,0 +1,30 @@
+function transpose(array) {
```

Comment:
What's this file for?

Reply:
I usually create a new file to write my solutions before moving it to the index.js.

I forgot to delete this particular one. 

I'm sorry about that, and I'll do proper checking next time before pushing.